src/Tribe/Views/V2/Hooks.php

```diff
@@ -60,6 +61,8 @@ protected function add_actions() {
 		add_action( 'tribe_events_pro_shortcode_tribe_events_after_assets', [ $this, 'action_disable_shortcode_assets_v1' ] );
 		add_action( 'updated_option', [ $this, 'action_save_wplang' ], 10, 3 );
 		add_action( 'the_post', [ $this, 'manage_sensitive_info' ] );
+		add_action( 'admin_bar_menu', [ $this, 'action_include_admin_bar_actions' ] );
+		add_action( 'parse_request', [ $this, 'action_process_suspending_cache_from_admin_bar' ] );
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 982e215cf5431c237bbd1d2533b7f5b1</sub>